build.sbt

```diff
@@ -1,11 +1,13 @@
-import BuildHelper._
+enablePlugins(ZioSbtCiPlugin, ScalafixPlugin)
+
+crossScalaVersions := Seq.empty
 
 inThisBuild(
   List(
-    organization := "dev.zio",
-    homepage := Some(url("https://zio.dev/zio-config/")),
-    licenses := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
-    developers := List(
+    name                       := "ZIO Config",
+    crossScalaVersions -= scala211.value,
+    ciEnabledBranches          := Seq("series/4.x"),
```

Comment:
I think we should include `master` as well, since `master` works with latest zio

Reply:
What about series/2.x? Doesn't it work for the latest version of ZIO?

Reply:
I don't think we need to support 2.x going forward